resources/views/main.blade.php

```diff
@@ -1,87 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Daskom Choose You</title>
-    <link rel="apple-touch-icon" sizes="180x180" href="{{ asset('/assets/favicon/apple-touch-icon.png') }}">
-    <link rel="icon" type="image/png" sizes="32x32" href="{{ asset('/assets/favicon/favicon-32x32.png') }}">
-    <link rel="icon" type="image/png" sizes="16x16" href="{{ asset('/assets/favicon/favicon-16x16.png') }}">
-    <link rel="stylesheet" type="text/css" href="{{ asset('/css/navbar.css') }}">
-    <link rel="stylesheet" type="text/css" href="{{ asset('/css/main.css') }}">
-    <link rel="stylesheet" type="text/css" href="{{ asset('/css/landing.css') }}">
-    <link rel="stylesheet" type="text/css" href="{{ asset('/css/reqruitment.css') }}">
-    <link rel="stylesheet" type="text/css" href="{{ asset('/css/about.css') }}">
-    <link rel="stylesheet" type="text/css" href="{{ asset('/css/footer.css') }}">
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet">
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"></script>
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@600&display=swap" rel="stylesheet">  
-    <link href="https://fonts.googleapis.com/css2?family=Electrolize&family=Fira+Code:wght@400&display=swap" rel="stylesheet">   
-    
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Daskom Choose You</title>
```

Comment:
Ingin memastikan, apakah itu memang tetap Daskom Choose You atau lupa diganti ke tagline tahun ini? @Soedipo

Reply:
Ah, benar itu lupa diganti tagline-nya

Reply:
nah itu kalau udah di fix, commit terus push aja ke branch yang sama ya @Soedipo ntar dia otomatis ke tambah di PR nya ini